### PR TITLE
Endpoint for column values

### PR DESCRIPTION
**Pull request by alexvetter**
This PR adds the functionally to retrieve all unique values of a shorttext column sorted by alphabet. It is only available for shorttext columns with languageType multi-language or language-neutral.

- `/tables/1/columns/1/values[/de]`

### CODE REVIEW

File: src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      _ <- sendRequest("POST", s"/tables/$tableId/columns", Columns(createBooleanColumnJson))
+        .map(_.getJsonArray("columns"))
+
+      _ <- sendRequest("GET", s"/tables/$tableId/columns/1/values/de") // should fail b/c multi-country is not allowed
+        .flatMap(_ => Future.failed(new Exception("this request should fail")))
+        .recoverWith({
+          case TestCustomException(_, _, 422) => Future.successful(())
+        })
+      _ <- sendRequest("GET", s"/tables/$tableId/columns/2/values/de") // should fail b/c shouldn't be called with langtag
+        .flatMap(_ => Future.failed(new Exception("this request should fail")))
```
Comment (Narigo) [inline src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala]: Maybe just use the comment as text in exception

File: src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        .map(_.getJsonArray("columns"))
+      _ <- sendRequest("POST", s"/tables/$tableId/columns", Columns(createShortTextColumnJson))
+        .map(_.getJsonArray("columns"))
+      _ <- sendRequest("POST", s"/tables/$tableId/columns", Columns(createNumericColumnJson))
+        .map(_.getJsonArray("columns"))
+      _ <- sendRequest("POST", s"/tables/$tableId/columns", Columns(createBooleanColumnJson))
+        .map(_.getJsonArray("columns"))
+
+      _ <- sendRequest("GET", s"/tables/$tableId/columns/1/values/de") // should fail b/c multi-country is not allowed
+        .flatMap(_ => Future.failed(new Exception("this request should fail")))
```
Comment (Narigo) [inline src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala]: Maybe just use the comment as text in exception

File: src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        .recoverWith({
+          case TestCustomException(_, _, 422) => Future.successful(())
+        })
+      _ <- sendRequest("GET", s"/tables/$tableId/columns/2/values/de") // should fail b/c shouldn't be called with langtag
+        .flatMap(_ => Future.failed(new Exception("this request should fail")))
+        .recoverWith({
+          case TestCustomException(_, _, 422) => Future.successful(())
+        })
+      _ <- sendRequest("GET", s"/tables/$tableId/columns/3/values") // should fail b/c numeric is not allowed
+        .flatMap(_ => Future.failed(new Exception("this request should fail")))
```
Comment (Narigo) [inline src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala]: Maybe just use the comment as text in exception

File: src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        .recoverWith({
+          case TestCustomException(_, _, 400) => Future.successful(())
+        })
+      _ <- sendRequest("GET", s"/tables/$tableId/columns/4/values") // should fail b/c boolean is not allowed
+        .flatMap(_ => Future.failed(new Exception("this request should fail")))
```
Comment (Narigo) [inline src/test/scala/com/campudus/tableaux/api/content/ColumnValuesTest.scala]: Maybe just use the comment as text in exception

**Comment by alexvetter** [comment]
:+1: Done!

**Pull request merged**